src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -310,8 +314,38 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
       .setHeaderActions(
         new LogOptionsScene({ visualizationType, onChangeVisualizationType: parentModel.setVisualizationType })
       )
-      .build();
-  }
+      .setOption('sortOrder', this.state.sortOrder)
+      .setOption('wrapLogMessage', this.state.wrapLogMessage)
+      .setOption('prettifyLogMessage', this.state.prettifyLogMessage);
+
+    if (!logsControlsSupported) {
+      panel.setOption('showTime', true);
+    } else {
+      panel
+        .setOption('showTime', getBooleanLogOption('showTime', true))
+        // @ts-expect-error Requires Grafana 12.1
+        .setOption('showControls', true)
+        // @ts-expect-error Requires Grafana 12.1
+        .setOption('controlsStorageKey', LOG_OPTIONS_LOCALSTORAGE_KEY)
```

Comment:
oh this is neat, but do we want to use the same key as used in Explore so folks get the same options set in both places?

Reply:
Yeah, let's do that. I can't find a good argument to keep them separated and it should help with going back and forth between modules.